src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelStateDictionary.cs

```diff
@@ -359,14 +361,30 @@ public void Merge(ModelStateDictionary dictionary)
         }
 
         /// <summary>
-        /// Sets the value for the <see cref="ModelState"/> with the specified <paramref name="key"/> to the
-        /// specified <paramref name="value"/>.
+        /// Sets the value for the <see cref="ModelState"/> with the specified <paramref name="key"/>.
```

Comment:
`value` is a bit vague here. Perhaps worth saying `Sets <see cref="ModelState.RawValue"> and <see cref="ModelState.AttemptedValue"> ...`